tsconfig.json

```diff
@@ -3,5 +3,7 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "jsxImportSource": "react"
-  }
+  },
+  "include": ["src/**/*.ts", "src/**/*.tsx", "src/**/*.astro"],
+  "exclude": ["node_modules", "src/pages/test.astro"]
```

Comment:
Why was this change needed, and why does it include `test.astro`, a file that doesn't exist?

Reply:
oh this was part of a separate edit- will remove